buildSrc/src/main/kotlin/com/datadog/gradle/utils/Version.kt

```diff
@@ -42,6 +42,10 @@ data class Version(
         object Snapshot : Type() {
             override val suffix: String = "-SNAPSHOT"
         }
+
+        object DevSnapshot : Type() {
+            override val suffix: String = "-dev-SNAPSHOT"
```

Comment:
you're missing a unit test for this case

Reply:
This is just a draft to publish this version, I don't plan to merge this.